EIPS/eip-5484.md

```diff
@@ -0,0 +1,101 @@
+---
+eip: 5484
+title: Consensual Soulbound Tokens
+description: Interface for special NFTS with immutable ownership and pre-determined immutable burn authorization
+author: Buzz Cai (@buzzcai)
+discussions-to: https://ethereum-magicians.org/t/eip-5484-consensual-soulbound-tokens/10424
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-8-17
+requires: 165, 721
+---
+
+
+## Abstract
+This EIP proposes another soulbound token standard. The interface is an extension of [EIP-721](./eip-721.md). Before issuance, both parties (the issuer and the receiver), have to agree on who has the authorization to burn this token. Burn authorization is immutable after declaration. After its issuance, soulbound tokens can't be transferred, but can be burned based on a predetermined immutable burn authorization.  
```

Comment:
You should write your EIP as a standalone document, so try not to reference other proposals unless you depend on them. Readers should be able to get the full context without having to look stuff up elsewhere.

```suggestion
This EIP defines an interface extending [EIP-721](./eip-721.md) to create soulbound tokens. Before issuance, both parties (the issuer and the receiver), have to agree on who has the authorization to burn this token. Burn authorization is immutable after declaration. After its issuance, a soulbound token can't be transferred, but can be burned based on a predetermined immutable burn authorization.  
```